packages/postgres-database/src/simcore_postgres_database/models/groups.py

```diff
@@ -0,0 +1,127 @@
+""" Groups table
+
+    - List of groups in the framework
+    - Groups have a ID, name and a list of users that belong to the group
+"""
+
+import enum
+
+import sqlalchemy as sa
+from sqlalchemy.sql import func
+
+from .base import metadata
+
+
+class GroupType(enum.Enum):
+    """
+        standard: standard group
+        primary: primary group
+        everyone: the only group for all users
+    """
+
+    STANDARD = "standard"
+    PRIMARY = "primary"
+    EVERYONE = "everyone"
+
+
+# NOTE: using func.now() insted of python datetime ensure the time is computed server side
+groups = sa.Table(
+    "groups",
+    metadata,
+    sa.Column("gid", sa.BigInteger, nullable=False, primary_key=True),
+    sa.Column("name", sa.String, nullable=False),
+    sa.Column("description", sa.String, nullable=False),
+    sa.Column("type", sa.Enum(GroupType), nullable=False, server_default="STANDARD"),
```

Comment:
should not server_default=GroupType.STANDARD?

Reply:
good point.

Reply:
both works.